tasks/dataset-split/main.py

```diff
@@ -0,0 +1,149 @@
+import csv
+import logging
+import zipfile
+
+from coretex import currentTaskRun, ImageDataset, CustomDataset, SequenceDataset, ImageSample, ImageDatasetClasses, CustomSample, SequenceSample
+from coretex.networking import NetworkRequestError
+
+from utils import splitOriginalSamples, SampleType, DatasetType
+
+
+def imageDatasetSplit(splitSamples: list[list[ImageSample]], datasetClasses: ImageDatasetClasses, projectID: int) -> list[ImageDataset]:
+    splitDatasets: list[ImageDataset] = []
+
+    for index, sampleChunk in enumerate(splitSamples):
+        splitDataset = ImageDataset.createDataset(f"{currentTaskRun().id}-splitDataset-{index}", projectID)
+        splitDataset.saveClasses(datasetClasses)
+
+        for sample in sampleChunk:
+            sample.unzip()
+            addedSample = splitDataset.add(sample.imagePath)
+            logging.info(f' >> [Dataset Split] The sample "{sample.name}" has been added to the dataset "{splitDataset.name}"')
```

Comment:
Use double quotes `"` instead of single quotest `'`. Remove the whitespace before `>>` in the string for all logs